modules/hooks/handlebars.js

```diff
@@ -27,11 +27,13 @@ export default function () {
         })
 
         Handlebars.registerHelper("tokenImg", function(actor) {
-            return actor.token ? actor.token.texture.src : actor.prototypeToken.texture.src
+            let token = game.canvas.tokens.getDocuments().filter(x=>x.actorId == actor.id)[0] ?? actor.prototypeToken;
+            return token.hidden ? "systems/wfrp4e/tokens/unknown.png" : token.texture.src;
         })
 
         Handlebars.registerHelper("tokenName", function(actor) {
-            return actor.token ? actor.token.name : actor.prototypeToken.name;
+            let token = game.canvas.tokens.getDocuments().filter(x=>x.actorId == actor.id)[0] ?? actor.prototypeToken;
```

Comment:
This is really gross but I think to make it cleaner you could just use `actor.getActiveTokens()`